tests/run-single-pg18-installcheck.sh

```
@@ -0,0 +1,802 @@
+# ---------------------------------------------------------------------------
+
+verify_subs_enabled() {
+	local node out any_bad=0
+	for node in ${NODES}; do
+		out="$(psql_on "${node}" -At -c \
+			"SELECT sub_name FROM spock.subscription WHERE NOT sub_enabled ORDER BY sub_name;" \
+			2>/dev/null)" \
+			|| { any_bad=1; log "${node}: NOT reachable -- treating as failure"; continue; }
+		if [ -n "${out}" ]; then
+			any_bad=1
+			log "${node}: DISABLED subscriptions: $(echo ${out} | tr '\n' ' ')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)